src/monitorService.ts

```diff
@@ -26,7 +26,7 @@ export class TxMonitorService {
     transactionTimeout: 180000
   };
 
-  constructor(wallet: ServerWeb3Wallet, logger=defaultLogger, options?: Partial<ITxMonitorOptions>) {
```

Comment:
I just noticed, I think you should move logger to options, otherwise this is breaking change and we need to bump major version